cypress/e2e/marc-authority/edit-marc-authority-save-keep-editing.cy.js

```diff
@@ -0,0 +1,242 @@
+      });
+    });
+  });
+
+  beforeEach('Visit MARC Authorities', () => {
+    cy.visit(TopMenu.marcAuthorities);
+    MarcAuthorities.waitLoading();
+    MarcAuthorities.searchBy(testData.authority.searchOption, testData.authority.searchInput);
+    MarcAuthorities.select(`${createdAuthorityIDs[0]}${authorityPostfix}`);
+    MarcAuthority.edit();
+  });
+
+  after('Delete test data', () => {
+    cy.getAdminToken();
+    createdAuthorityIDs.forEach((id) => {
+      MarcAuthority.deleteViaAPI(id);
+    });
+    Users.deleteViaApi(testData.userProperties.userId);
+  });
+
+  it(
+    'C360092 Verify that click on the "Save & keep editing" button doesnt close the editing window of "MARC Authority" record (spitfire) (TaaS)',
+    { tags: [TestTypes.extendedPath, DevTeams.spitfire] },
+    () => {
+      // Verify initial state of edit view
+      QuickMarcEditor.checkHeaderFirstLine(
+        headerContent.initialHeaderContent.marcData,
+        headerContent.initialHeaderContent.source,
+      );
+      QuickMarcEditor.checkButtonsDisabled();
+
+      // Edit record and verify button states updated
+      MarcAuthority.changeField(
+        testData.editedFields[0].tag,
+        `${subfieldPrefix}${testData.editedFields[0].content}`,
+      );
+      QuickMarcEditor.checkButtonsEnabled();
+
+      // Save edits and verify view updated
+      QuickMarcEditor.clickSaveAndKeepEditing();
+      QuickMarcEditor.checkButtonsDisabled();
+      QuickMarcEditor.checkHeaderFirstLine(
+        headerContent.editedHeaderContent.marcData,
+        headerContent.editedHeaderContent.source,
+      );
+
+      // Add field and verify button states updated
+      MarcAuthority.addNewField(4, testData.newField.tag, testData.newField.content);
+      QuickMarcEditor.checkButtonsEnabled();
+
+      // Save added field and verify view updated
+      QuickMarcEditor.clickSaveAndKeepEditing();
+      QuickMarcEditor.checkButtonsDisabled();
+      QuickMarcEditor.checkHeaderFirstLine(
+        headerContent.editedHeaderContent.marcData,
+        headerContent.editedHeaderContent.source,
+      );
+
+      // Delete added field and verify states
+      cy.wait(4000);
+      QuickMarcEditor.deleteFieldByTagAndCheck(testData.newField.tag);
+      QuickMarcEditor.checkButtonsEnabled();
+
+      // Save deletion and verify modal
+      QuickMarcEditor.clickSaveAndKeepEditing();
+      QuickMarcEditor.checkDeleteModal(1);
+      QuickMarcEditor.confirmDelete();
+
+      // // Confirm deletion and verify view updated
+      QuickMarcEditor.checkFieldAbsense();
+      QuickMarcEditor.checkButtonsDisabled();
+      QuickMarcEditor.checkHeaderFirstLine(
+        headerContent.editedHeaderContent.marcData,
+        headerContent.editedHeaderContent.source,
+      );
+
+      // Restore deleted field and verify states
+      QuickMarcEditor.deleteFieldByTagAndCheck(testData.deletedField.tag);
+      QuickMarcEditor.clickRestoreDeletedField();
```

Comment:
No assertion from Step13
The placeholder messages have been replaced by deleted rows with same values as it displayed before user clicked on the "Delete this field" icons.